lib/paper_trail/reifier.rb

```diff
@@ -284,9 +282,8 @@ def reify_has_many_through(transaction_id, associations, model, options = {})
       # table, this method would return the constant `Fruit`.
       def version_reification_class(version, attrs)
         inheritance_column_name = version.item_type.constantize.inheritance_column
-        class_name = attrs[inheritance_column_name].blank? ?
```

Comment:
Do people ever set `inheritance_column` to `""` ? That's the only case in which this would break...

Happy to revert this change if needed. Looking through `git`, it's been `blank?` since the code was first written in 2009 (so not in response to a bug report from someone using `inheritance_column = ""`).